src/mlpack/core/util/timers.hpp

```diff
@@ -15,7 +15,11 @@
 
 #include <map>
```

Comment:
Can you add your name to the authors list and also adjust ``MLPACK``?

Reply:
Good catch, done.